[DOC] Docs update to add installing perses from source section. (#1679)
perses/perses: docs/user-guides/installing-perses-from-source.md

This sections continues with installing using the source code, test the project, build the project, configure examples,
and run the Perses dashboard project on your local machine.

## Building requirements

To build the Perses project from source you need minimum versions of the following:

> - GO v1.18+

Author: Fixed and pushed in branch.

- Node v16+
- NPM v8+

## Downloading the source

You can obtain the source code of the Perses project at the main product page in various compression formats or you can
clone the git repository as shown here:

```shell
git clone https://github.com/perses/perses.git
```

> Now just change into the Perses project root directory and build the project:

Reviewer: ```suggestion
Now move to the perses folder and build the project
```

Author: Fixed in branch.

```shell
cd perses

make build

...
... LOTS OF BUILD LOG LINES HERE ...
...
@perses-dev/app:build: webpack 5.84.1 compiled with 2 warnings in 7038 ms

  Tasks:    14 successful, 14 total
  Cached:    0 cached, 14 total
  Time:    28.59s

>> compressing assets
scripts/compress_assets.sh
GOARCH=arm64 GOOS=darwin go generate ./internal/api
>> build the perses api
CGO_ENABLED=0 GOARCH=arm64 GOOS=darwin go build -ldflags "-s -w -X github.com/prometheus/common/version.Version=0.41.1
>> build the perses cli
CGO_ENABLED=0 GOARCH=arm64 GOOS=darwin go build -ldflags "-s -w -X github.com/prometheus/common/version.Version=0.41.1

> Perses server built successfully!

Author: Fixed in branch.

```

Note the last line stating that you have successfully built your own instance of Perses!

## Starting the Perses server

> To start the server you need a minimal configuration file, so let's set up a new one for you. Apply the following to a

Reviewer: That's actually not true, Perses can be started without providing any configuration file.

I think you shouldn't give a configuration like that, there is a high chance it doesn't match what users are looking for.

Probably you should just point to the doc about the configuration. Each feature should require a proper documentation that will contain the proper configuration.
From my point of view it's ok to skip this section. The scope of this part is just to build Perses from source. Configuring Perses is something else.

Author: Fixed in branch.

file using your favorite editor and name it myconfig.yaml:

```yaml
database:
  file:
    folder: "myperses/local_db"
    extension: "json"

provisioning:
  folders:
  - "myperses/data"

schemas:
  panels_path: "schemas/panels"
  queries_path: "schemas/queries"
  datasources_path: "schemas/datasources"
  variables_path: "schemas/variables"
  interval: "5m"
```

The database section points to a new directory called `myperses/local_db` where all your visualization and dashboard
project files will be kept. The provisioning sections defines where all the JSON files will be stored, in
`myperses/data`. The schema section defines where Perses can find the definitions for all things used to create
visualization and dashboard elements.

Once you have the configuration file set up, you can now start your Perses instance:

```shell

> ./bin/perses -config [PATH_TO]/myconfig.yaml

Reviewer: ```suggestion
./bin/perses
```

Author: Fixed in branch.
